libraries/hermes/src/utils/ProtoGenerator.ts

```diff
@@ -44,6 +50,11 @@ export class ProtoGenerator {
           r as SocketProtoDescription,
           protoFunctions,
         );
+      } else if (r.hasOwnProperty('proxy')) {
```

Comment:
if proxy exists do not add anything to protoFunctions, there is no proto in proxy